_studio/mfx_lib/vpp/src/mfx_vpp_sw_core.cpp

```diff
@@ -1211,17 +1211,21 @@ mfxStatus VideoVPPBase::Reset(mfxVideoParam *par)
         }
     }// Opaque
 
+    bool isCompositionModeEnabledInNewParams = (MFX_ERR_NONE == GetCompositionEnabledStatus(par)) ? true : false;
```

Comment:
why do you use ?: here to convert bool into bool?
operator == already returns bool

Reply:
check what GetCompositionEnabledStatus returns